### DISCUSSION TITLE
Serving cached API responses?

### DISCUSSION THREAD

**Discussion by aleksb3** [webstudio-is/webstudio-community#147] [Q&A]
Is it possible to have a fallback so that if a live webstudio site tries to hit an API that is down to serve a previously cached version of the response? Alternatively would it be possible to serve the webstudio site as a static site that only pulls content from an API in when the site is initially published?

**Comment by kof** [comment]
Both questions are a "no" at the moment, responses are cached, but the cache is not a guarantee, it's a speed improvement.
